src/Common/Core/Impl/Collections/EnumerableExtensions.cs

```diff
@@ -113,5 +113,11 @@ public static class EnumerableExtensions {
                 }
             }
         }
+
+        public static void ForEach<T>(this IEnumerable<T> enumerable, Action<T> action) {
```

Comment:
But it isn't needed in your changeset, where can use either `AddRange` or `ToList` depending on the preferred style, but either of these options gives you better code.

Many of the design patterns are 'philosophical objections', yet I don't want to add an extension that doesn't give any advantage so far.